cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -0,0 +1,154 @@
+import {Organization} from '../../../src/types'
+
+describe('FluxQueryBuilder', () => {
+  beforeEach(() => {
+    cy.flush()
+    cy.signin()
+    cy.get('@org').then(({id}: Organization) => {
+      cy.fixture('routes').then(({orgs, explorer}) => {
+        cy.visit(`${orgs}/${id}${explorer}`)
+        cy.getByTestID('tree-nav').should('be.visible')
+        // Switch to Flux Query Builder
+        cy.setFeatureFlags({newDataExplorer: true})
```

Comment:
I believe this is a new pattern for flux query builder 😄 Do you mean to set the feature flag on in `before` function instead of in `beforeEach` function?